closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -261,13 +266,44 @@ static void printErrors(void)
     fflush(stderr);
 }
 
+/*
+ * We use a 8 digit map (ABBCCDDD) to represent the version of openssl.
+ * A is the major version,
+ * BB is the minor version,
+ * CC is the revision,
+ * DDD for the letter that could be present in any version.
+ * For example, if an openssl version is in this scheme 1.2.3.d
+ * where major is 1, minor is 2, reversion is 3 and last char is d -> 4.
+ * So the result would be 0x10203004.
+ */
+static jlong extractVersionToJlong(const char *astring)
+{
+    long major = 0;
+    long minor = 0;
+    long revision = 0;
+    long diff = 0;
+    char last_char = 0;
```

Comment:
Please rename these for consistency with the new definition of `OPENSSL_VERSION_CODE` (and update the documentation above accordingly).

Reply:
Updated.